### PR TITLE
New time picker

### PR DESCRIPTION
**Pull request by thilan-tran**
Added a new time picker with dropdown fields + input user can type into.

input_time_picker

### CODE REVIEW

**Review by claireg22 — APPROVED** [review]
Looks good!

File: src/components/presentational/Task.js [hunk 1]
```diff
@@ -189,17 +207,13 @@ const Task = ({
...
-        disabled={!editing}
-      />
-      <Label editing={editing}>Category:</Label>
-      <Dropdown
-        selected={category}
-        onSelect={(option) => setCategory(option)}
-        options={categories}
+        value={new Date(due).toISOString()}
+        onChange={(e) => {
+          console.log('task', e.target.value);
```
Comment (alexlongerbeam) [inline src/components/presentational/Task.js]: Remove before merging

File: src/components/presentational/Dropdown.js [hunk 2]
```diff
@@ -352,6 +366,398 @@ const months = [
...
+            // setDate(e.target.value);
+            handleChange(e.target.value, time);
+          }}
+        />
+        <TimePicker
+          disabled={disabled}
+          placeholder="Time"
+          value={time}
+          onChange={(e) => {
+            // setTime(e.target.value);
```
Comment (alexlongerbeam) [inline src/components/presentational/Dropdown.js]: remove before merging

File: src/components/presentational/Dropdown.js [hunk 3]
```diff
@@ -352,6 +366,398 @@ const months = [
...
+
+  return (
+    <DateTimeWrapper>
+      <Row spaceEvenly style={{ width: '100%', marginLeft: '-8px' }}>
+        <DatePicker
+          disabled={disabled}
+          placeholder={placeholder}
+          value={date.toDateString()}
+          onChange={(e) => {
+            // setDate(e.target.value);
```
Comment (alexlongerbeam) [inline src/components/presentational/Dropdown.js]: remove

File: src/components/presentational/Dropdown.js [hunk 4]
```diff
@@ -352,6 +366,398 @@ const months = [
...
+    setDate(new Date(value));
+    scrollRef(hourRef, 80);
+    scrollRef(minuteRef, 80);
+
+    // hourRef.current.parentNode.scrollTop =
+    //   hourRef.current.offsetTop - hourRef.current.parentNode.offsetTop - 100;
+    // minuteRef.current.parentNode.scrollTop =
+    //   minuteRef.current.offsetTop -
+    //   minuteRef.current.parentNode.offsetTop -
+    //   100;
```
Comment (alexlongerbeam) [inline src/components/presentational/Dropdown.js]: remove

File: src/components/presentational/Dropdown.js [hunk 5]
```diff
@@ -242,6 +239,20 @@ const DropdownNumpadGrid = styled.div`
...
+//   return (
+//     <Dropdown
+//       selected={props.val}
+//       onSelect={props.setVal}
+//       options={nums}
+//       disabled={props.hidden}
+//     />
+//   );
+// };
+
```
Comment (alexlongerbeam) [inline src/components/presentational/Dropdown.js]: Remove commented code before merging if it's not in use

**Review by alexlongerbeam — APPROVED** [review]
Looks good! Approving to unblock. Before merging though, please remove the comments and console.log so those don't get stuck in master

**Pull request merged**